android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -804,6 +804,16 @@ class XMTPModule : Module() {
             }
         }
 
+        AsyncFunction("addedByAddress") Coroutine { clientAddress: String, id: String ->
+            withContext(Dispatchers.IO) {
+                logV("addedByAddress")
+                val client = clients[clientAddress] ?: throw XMTPException("No client")
```

Comment:
I know you were just copying the pattern in other functions but this also happens inside findGroup so is redundant. You can just remove it.
```suggestion
```